frontend/packages/topology/src/components/quick-search/QuickSearchList.tsx

```diff
@@ -69,7 +71,12 @@ const QuickSearchList: React.FC<QuickSearchListProps> = ({
             className={cx('odc-quick-search-list__item', {
               'odc-quick-search-list__item--highlight': item.uid === selectedItemId,
             })}
-            onDoubleClick={(e: React.SyntheticEvent) => handleCta(e, item, closeModal)}
+            onDoubleClick={(e: React.SyntheticEvent) => {
+              handleCta(e, item, closeModal);
+              fireTelemetryEvent('Quick Search used', {
+                item: item.name,
```

Comment:
We should include additional info about the item:
```suggestion
              fireTelemetryEvent('Quick Search Used', {
                id: item.uid,
                type: item.type,
                name: item.name,
```